### PR TITLE
Fix conditional render logic to display error string to user

### LINKED ISSUE #143: Searched conditional rendering not working as expected
## Expected Behavior
If user searched invalid name or number, page should show a string telling user to search a valid pokemon name.

## Actual/Current Behavior
No string.

## Steps to Reproduce the Problem

  1. Search random string.
  2. Observe.
  3. 

## Pre-Testing TODOs

  1. Change fetch or conditional.

### If you are not a member of this project, _skip this step_

  1. Search random string.
  2. Observe.

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Change PokeList conditional rendering.

## Purpose
Get error string to be displayed if user searched invalid pokemon.

## Approach
Change conditional render logic to display error string to user if they enter invalid pokemon name or dex number.

## Testing Steps
Search random string or number. Expect a h3 tag to be displayed.

Closes #143

### CODE REVIEW

File: src/components/PokeApp.js [hunk 1]
```diff
@@ -22,13 +22,13 @@ const PokeApp = () => {
...
-  // Function for single pokemon fetch, invoked in Header
+  // Function for single pokemon fetch(by name or dex number), invoked in Header
   const getSinglePokemon = async (name) => {
     try {
       // call fetch api function
       const pokemonData = await fetchPokemon(name);
-      if (!pokemonData.length) {
-        setSinglePokemon([pokemonData]);
+      if (pokemonData) {
+        setSinglePokemon(pokemonData);
```
Comment (BlackCubes) [inline src/components/PokeApp.js:31]: Good catch

File: src/components/PokeList.js [hunk 2]
```diff
@@ -223,13 +223,13 @@ const PokeList = ({
         ) // end react fragment
       ) : (
         // show single user search
-        singlePokemon[0] && (
+        singlePokemon? (
```
Comment (BlackCubes) [inline src/components/PokeList.js:226]: Nice, nice

**Review by BlackCubes — APPROVED** [review]
Good fixes!

**Review by Lap343 — APPROVED** [review]
Very nice

**Pull request merged**